gator/checks/check_ListChecks.py

```diff
@@ -0,0 +1,98 @@
+"""List the internal and external checks available through pluginbase."""
```

Comment:
I'm not sure that this should be a check itself -- it does fit in, but semantically it's very different, and so we might, in the future, want to move it up to the core functionality of GatorGrader.  I think it's a bad idea to start building too much core functionality into plugins.

Reply:
After looking through everything, I don't think there's a better alternative to having this here, but it still doesn't feel right.

Reply:
Yeah, I agree, and I've felt this way from the start of this PR. In the short term, I also can't quickly think of a better way to provide this functionality so I've made a note of this point and I will soon raise it as an issue. Thanks for pointing out this concern, it is a good one!